virtcontainers/kata_agent.go

```diff
@@ -831,7 +832,10 @@ func (k *kataAgent) appendDevices(deviceList []*grpc.Device, c *Container) []*gr
 			ContainerPath: dev.ContainerPath,
 		}
 
-		if d.SCSIAddr == "" {
+		if d.VirtPath != "" {
```

Comment:
Open for suggestions here on how to figure out the actual type. Today we have no top level type information.

Reply:
This has been fixed by deriving the information directly.